app/models/question.rb

```diff
@@ -12,6 +12,10 @@ def with_answers
       includes(:answers)
     end
 
+    def top
+      where("votes > ?", 0).order("votes DESC")
```

Comment:
Hmmm ... 
I don't think there is any need for the `where` condition we are more interested in the `ordering` of the votes e.g

> order("votes DESC")